src/BenchmarksDriver/Program.cs

```diff
@@ -724,9 +724,20 @@ private static Task WriteJobsToSql(ServerJob serverJob, ClientJob clientJob, str
                         [Path] [nvarchar](max) NULL,
                         [Method] [nvarchar](max) NOT NULL,
                         [Headers] [nvarchar](max) NULL,
-                        [Dimension] [nvarchar](max) NOT NULL,
+                        [Dimension] [nvarchar](64) NOT NULL,
                         [Value] [float] NOT NULL
                     )
+
+                    CREATE INDEX IDX_Scenario_Hardware_OS_Description_Dimension
+                    ON AspNetBenchmarks
+                    (
+                        [Scenario],
+                        [Hardware],
+                        [OperatingSystem],
+                        [Description],
```

Comment:
Why index `Description`?

Reply:
This is to get only the data points that are run with the CI job. If we get more automatic data points like 2.0/2.1 comparisons, we don't want these to be in the graphs.

Reply:
BTW I have another branch (not yet PR'ed) to set this description as part of the CI job.